### PR TITLE
Header

### PR DESCRIPTION
**Pull request by reionut**

### CODE REVIEW

File: complete/style.css [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+}
+
+.logo {
+  padding: 14px 16px;
+  flex: 1;
+}
+/* === TOP NAVBAR STYLE === */
+
+.top-navbar {
+  flex:2;
```
Comment (catalinberta) [inline complete/style.css]: header-menu

When resizing the browser, the behavior of the menu is not as how we pictured it.
Here, in this image, it looks like the button `contact us` would have room to fit in there, yet it shows on a second line?

Here is our ideal behavior of the header when resizing the browser smaller:
- the menu is *always* on one line
- when there is no more room, remove the social icons completely
- if even afterwards there is no more room, then stack everything vertically (with the logo AT the top)
- when the menu is vertical, it should have 100% width

HINT: The current issue is due to using flex: 1 for logo and social icons and flex 2 for the menu.
We would recommend removing them and only add flex 1 to the menu

File: complete/index.html [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+      <div class="top-nav-social">
+      <a href="#" class="fa fa-facebook"></a>
+      <a href="#" class="fa fa-twitter"></a>
+      <a href="#" class="fa fa-skype"></a>
+      <a href="#" class="fa fa-linkedin"></a>
+      </div>
+      </div>
+    <div class="header-text">
+        <h1>home</h1>
+        <h3>power by psdfreebies.com</h3>
```
Comment (catalinberta) [inline complete/index.html:35]: letter-spacing

The text `power by psdfreebies.com` is not like Photoshop, all spaced out and blue.
_HINT: there is a css property to space out letters in a text_

File: complete/style.css [hunk 3]
```diff
@@ -0,0 +1,155 @@
+/* === HEADER STYLE === */
+
+.header {
+  background: url(images/header-bg.png) no-repeat center;
+  background-size: cover;
+  width: 100wv;
```
Comment (catalinberta) [inline complete/style.css]: This is wrong, it's `vw`, but also it does not help with anything here

File: complete/index.html [hunk 4]
```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="style.css" />
+  <script src="script.js"></script>
+  <meta name="description" content="Responsive Website">
+  <title>Responsive Website</title>
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/meyer-reset/2.0/reset.css">
```
Comment (catalinberta) [inline complete/index.html]: reset.css is `never` after the other stylesheets, it's always first, so then the stylesheets we (you) write can overwrite the reset.css values when needed

Comment (catalinberta) [reply complete/index.html]: reset.css, libraries and all other external assets need to be added before, so you can then code your own code on top of them

File: complete/index.html [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+      <li><a href="#pages">Pages</a></li>
+      <li><a href="#features">Features</a></li>
+      <li><a href="#extensions">Extensions</a></li>
+      <li><a href="#tutorials">Tutorials</a></li>
+      <li><a href="#contact_us">Contact us</a></li>
+      </ul>
+      </nav>
+      <div class="top-nav-social">
+      <a href="#" class="fa fa-facebook"></a>
+      <a href="#" class="fa fa-twitter"></a>
```
Comment (catalinberta) [inline complete/index.html:28]: social

Can we make the icons have a subtle highlight when hovering with the mouse? see picture with hover on the twitter icon

File: complete/index.html [hunk 6]
```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="style.css" />
+  <script src="script.js"></script>
+  <meta name="description" content="Responsive Website">
```
Comment (catalinberta) [inline complete/index.html]: non-closing elements need the `/>` at the end.

File: complete/style.css (hunk 1 above)
Comment (catalinberta) [reply complete/style.css]: Here are some more exampls of an ideal behavior

menu2


menu3

**Review by catalinberta — COMMENTED** [review]
Done reviewing!

**Comment by reionut** [comment]
Fixed everything at the Header

**Pull request merged**